lib/policies/key-auth/index.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
-  policy: require('./keyauth'),
+  policy: require('./key-auth'),
```

Comment:
The policy file is still named `keyauth.json`.

Reply:
![image](https://user-images.githubusercontent.com/1416224/43981587-bc4a3004-9cf2-11e8-8068-6ef24bd1ec4e.png)
It has been renamed @kevinswiber — the Github diff doesn't really show it that clearly enough

Reply:
Also – in case tests would pass with such thing, I'd really go and find another job 😱

Reply:
@XVincentX Oh, my bad.  I was browsing the code at the commit.  But instead of picking the last commit, I picked the first one, which had not included these changes.  We're good!